dist/amd/aurelia-binding.js

```diff
@@ -47,7 +47,7 @@ define(['exports', 'aurelia-logging', 'aurelia-pal', 'aurelia-task-queue', 'aure
   var _typeof = typeof Symbol === "function" && typeof Symbol.iterator === "symbol" ? function (obj) {
     return typeof obj;
   } : function (obj) {
-    return obj && typeof Symbol === "function" && obj.constructor === Symbol ? "symbol" : typeof obj;
```

Comment:
@EisenbergEffect, adding files under dist to the PR is a recurring problem. Perhaps this check could be automated, for example using custom client-side git hook (that could be installed with npm postinstall script)